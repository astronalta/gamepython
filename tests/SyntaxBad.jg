Blarg < Object {

    func1() {
        a mutable = 0

        2001! bb
        ~` ejd
        ~3 += 3;
        @init() {
        }
        for x im h {
            a = 0
            @
        }
        foo[] = 9
        a = 92
    }

    Int func3() {
    }

    func3() {
        `
        if q {
        } 
    }

    Int a
    blarg += 39
    b : Int
    
    func2() Int {
}
