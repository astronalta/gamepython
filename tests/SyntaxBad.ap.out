SyntaxBad.ap:6:13: Syntax error, unexpected '!'
SyntaxBad.ap:7:10: Syntax error, unexpected BACKTICK
SyntaxBad.ap:8:12: Syntax error, unexpected ADD_ASSIGN
SyntaxBad.ap:9:9: Syntax error, unexpected OPERATOR
SyntaxBad.ap:11:15: Syntax error, unexpected IDENTIFIER, expecting IN
SyntaxBad.ap:13:13: Syntax error, unexpected $undefined
SyntaxBad.ap:15:13: Syntax error, unexpected ']'
SyntaxBad.ap:19:5: Syntax error, unexpected TYPE, expecting IDENTIFIER or OPERATOR or '}'
SyntaxBad.ap:23:9: Syntax error, unexpected BACKTICK
SyntaxBad.ap:28:5: Syntax error, unexpected TYPE, expecting IDENTIFIER or OPERATOR or '}'
SyntaxBad.ap:29:11: Syntax error, unexpected ADD_ASSIGN, expecting '=' or TYPE or TYPEVAR
SyntaxBad.ap:30:7: Syntax error, unexpected ':', expecting '=' or TYPE or TYPEVAR
SyntaxBad.ap:34:2: Syntax error, unexpected $end, expecting IDENTIFIER or OPERATOR or '}'
